views/includes/metadata.njk

```diff
@@ -1,58 +1,67 @@
-<link rel="apple-touch-icon" sizes="180x180" href="/assets/images/icons/apple-icon-180x180.png">
-<link rel="icon" type="image/png" sizes="192x192"  href="/assets/images/icons/android-icon-192x192.png">
-<link rel="icon" type="image/png" sizes="32x32" href="/assets/images/icons/favicon-32x32.png">
-<link rel="icon" type="image/png" sizes="96x96" href="/assets/images/icons/favicon-96x96.png">
-<link rel="icon" type="image/png" sizes="16x16" href="/assets/images/icons/favicon-16x16.png">
-<meta name="msapplication-TileImage" content="/assets/images/icons/ms-icon-144x144.png">
-
-{% if isBilingual %}
-    {% if locale == 'en' %}
-        <link rel="alternate" hreflang="cy" href="{{ getCurrentAbsoluteUrl('cy') }}" />
-    {% else %}
-        <link rel="alternate" hreflang="en" href="{{ getCurrentAbsoluteUrl('en') }}" />
-    {% endif %}
-{% endif %}
+{%- set metaTitle = getMetaTitle(globalCopy.brand.title, title) -%}
+{%- set baseDescription = openGraph.description if openGraph.description else description -%}
+{%- set metaDescription = baseDescription | default(globalCopy.brand.description, true) %}
+        <!-- deploy {{ appData.deployId }} | build {{ appData.buildNumber }} | node version {{ appData.nodeVersion }} -->
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
+
+        <meta name="title" content="{{ metaTitle }}">
+        <meta name="description" content="{{ metaDescription }}">
+
+        {# Open Graph data #}
+        <meta property="og:type" content="website">
+        <meta property="og:url" content="{{ getCurrentAbsoluteUrl() }}">
+        <meta property="og:title" content="{{ openGraph.title if openGraph.title else metaTitle }}">
+        <meta property="og:description" content="{{ metaDescription }}">
+        {% if openGraph.images.facebook -%}
+            <meta property="og:image" content="{{ openGraph.images.facebook | safe }}">
+        {% elseif socialImage -%}
+            <meta property="og:image" content="{{ getSocialImageUrl(socialImage) | safe }}">
+        {% endif %}
+
+        <meta name="twitter:site" content="@{{ globalCopy.brand.twitter }}" />
+        {% if socialImage or openGraph.images.twitter or openGraph.images.facebook -%}
+            <meta name="twitter:card" content="summary_large_image" />
+        {% else -%}
+            <meta name="twitter:card" content="summary" />
+        {% endif -%}
+
+        {% if openGraph.images.twitter -%}
+            <meta property="twitter:image" content="{{ openGraph.images.twitter | safe }}">
+        {% elseif socialImage -%}
+            <meta property="twitter:image" content="{{ getSocialImageUrl(socialImage) | safe }}">
+        {% endif %}
+
+        <meta name="google-site-verification" content="YJm5YK8XXPjyF0oyks3mvJ2P2qc1mwm9GyF7OlyNl9A" />
```

Comment:
Everything below this is just a whitespace change – I changed the indentation so all of these fields look right, eg. not like this:

![image](https://user-images.githubusercontent.com/394376/54542694-35510380-4994-11e9-95e8-6f341e12a51c.png)

Reply:
Why does indentation of the output matter? I don't feel that strongly about this, just curious.

Reply:
It doesn't really, it just made it easier to scan it by eye when checking for updates etc.